File: frontend/src/app/dto/learning-material/learning-material.insert.res.dto.ts
import { LearningMaterialInsertDataResDto } from "./learning-material.insert.data.res.dto";

export class LearningMaterialInsertResDto{
    private _data! : LearningMaterialInsertDataResDto
	private _message! : string

    public get data(): LearningMaterialInsertDataResDto {
        return this._data;
    }

    public set data(data: LearningMaterialInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}

File: backend/src/main/java/com/lawencon/elearn/dto/learningmaterial/LearningMaterialInsertResDto.java
package com.lawencon.elearn.dto.learningmaterial;

public class LearningMaterialInsertResDto {
	private LearningMaterialInsertDataResDto data;
	private String message;
	public LearningMaterialInsertDataResDto getData() {
		return data;
	}
	public void setData(LearningMaterialInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
